### ISSUE TITLE
translate pytorch version of cyclegan to keras, loss graph seems to be weird

### ISSUE THREAD

**Issue by BeomSooKim** [junyanz/pytorch-CycleGAN-and-pix2pix#688]
I write keras version of cyclegan based on your pytorch-cyclegan
but loss seems to be wrong.
cycle loss is decreasing over the epochs. and at around 100 epochs, it becomes 0.
but discriminator loss and adversarial loss are decrease slightly until about 30 epochs, after that they saturated near 0.4~0.5.
If i add identity loss, it moves like cycle loss and result is still weird.

I trained your code and plotted loss curve. on your model, D loss is fall down after 20 epochs, while adversarial loss incerease, and then after 30 epochs, D loss increases and adversarial loss is falling down. 

I think the problem is that D is not trained well but i don't know exactly.
I used summer2winter_yosemite dataset and all parameter setting is the same your paper. 
(sorry for my English, I can't speak english well)

**Comment by junyanz** [comment]
I am not sure why. You can look at existing CycleGAN Keras [implementation](https://github.com/eriklindernoren/Keras-GAN/blob/master/cyclegan/cyclegan.py) for a reference.
